packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_typedefs.hpp

```diff
@@ -64,6 +65,7 @@ typedef typename Tpetra::Map<>             map_t;
 typedef typename Tpetra::CrsGraph<>        graph_t;
 typedef typename Tpetra::CrsMatrix<Scalar> matrix_t;
 typedef typename Tpetra::Export<>          export_t;
+typedef typename Tpetra::MultiVector<Scalar> multivector_t;
 
 // NOTE: Arrays are hardwired for QUAD4
```

Comment:
See note above (just stick a constexpr `dof_per_elt` in this file & you're done :D ).